modules/prebidServerBidAdapter/index.js

```diff
@@ -366,10 +366,57 @@ let nativeEventTrackerMethodMap = {
  */
 let bidIdMap = {};
 let nativeAssetCache = {}; // store processed native params to preserve
+
+/**
+ * map wurl to auction id and adId for use in the BID_WON event
+ */
+const wurlMap = {};
+
+/**
+ * @param {string} auctionId
+ * @param {string} adId generated value set to bidObject.adId by bidderFactory Bid()
+ * @param {string} wurl events.winurl passed from prebidServer as wurl
+ */
+function addWurl(auctionId, adId, wurl) {
+  if ([auctionId, adId].every(utils.isStr)) {
+    wurlMap[`${auctionId}${adId}`] = wurl;
+  }
+}
+
+/**
+ * @param {string} auctionId
+ * @param {string} adId generated value set to bidObject.adId by bidderFactory Bid()
+ */
+function removeWurl(auctionId, adId) {
+  if ([auctionId, adId].every(utils.isStr)) {
+    wurlMap[`${auctionId}${adId}`] = undefined;
+  }
+}
+/**
+ * @param {string} auctionId
+ * @param {string} adId generated value set to bidObject.adId by bidderFactory Bid()
+ * @return {(string|undefined)} events.winurl which was passed as wurl
+ */
+function getWurl(auctionId, adId) {
+  if ([auctionId, adId].every(utils.isStr)) {
+    return wurlMap[`${auctionId}${adId}`];
+  }
+}
+
+/**
+ * remove all cached wurls
+ */
+export function resetWurlMap() {
```

Comment:
is this used?

Also, why not just 

`wurlMap = {};`?

Reply:
@idettman Bump

Reply:
it's only used by the tests to clear state between tests, the reset function is used to avoid exposing `wurlMap` for the module, but now I'm thinking it's not worth the extra kb.